### PR TITLE
Elastic search query dsl bug fix

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-19716)

## Description
Fixed an issue in the **search** and **es-search** commands the index parameter was always taken from integration parameters rather than the index argument to those commands.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed UT - https://app.circleci.com/pipelines/github/demisto/content/218789/workflows/7c615227-d02f-4066-996a-cb6601c825ca/jobs/620135

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py [hunk 1]
```diff
@@ -915,3 +915,25 @@ def json(self):
...
+    When
+    - executing the search command
+
+    Then
+     - make sure that the index is being taken from the command arguments and not from integration parameters
+    """
+    import Elasticsearch_v2
+    Elasticsearch_v2.FETCH_INDEX = 'index from parameter'
+    mocker.patch.object(
+        demisto, 'args', return_value={'index': 'index from arg', 'query_dsl': 'test', 'size': '5', 'page': '0'}
```
Comment (JudahSchwartz) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py:934]: id extrcact "index from arg" to an argument

**Review by JudahSchwartz — APPROVED** [review]
Nice!

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py (hunk 1 above)
Comment (GuyAfik) [reply Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py]: done :)

**Pull request merged**